src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$base-color: #060b35;
+$secondary-color: #616070;
+$button-buy-color: #00acdc;
+$button-buy-activecolor: #fff;
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)

when all elements have the same `box-sizing` value, their behavior becomes more predictable

P.S  after fixing this comment remove box-sizing form `card` and `__buy-button:hover`